src/fragment-components/a-numberinput.tsx

```diff
@@ -233,23 +233,24 @@ export const componentInfo: ComponentInfo = {
 			imports: ['NumberInput'],
 			code: ({ json }) => {
 				return `<NumberInput
+					id="${json.id}"
```

Comment:
Why is this needed? Should it be a string or a number?

Reply:
Its required prop of number input and it should be a string i believe. 
<img width="680" alt="Screenshot 2023-03-10 at 12 00 12 PM" src="https://user-images.githubusercontent.com/23690711/224377448-2674a573-3e9c-43e8-a458-ffe0199991d5.png">

<img width="712" alt="Screenshot 2023-03-10 at 12 01 12 PM" src="https://user-images.githubusercontent.com/23690711/224377456-b258ee4c-a562-4e62-b56a-4893af9e828d.png">